converter/fluka/cards/assignmat_card.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass, field
+from converter.fluka.cards.card import Card
+from converter.fluka.helper_parsers.material_parser import MaterialAssignment
+
+
+@dataclass
+class AssignmatsCard:
+    """
+    Class representing description of material assignment in FLUKA input.
+    Every material assignment is represented by one line:
+        codwed - "ASSIGNMA"
```

Comment:
what is `codewd`  ? "code word" ?

Reply:
I've changed it to match description in FLUKA:
https://flukafiles.web.cern.ch/manual/chapters/general_features_of_fluka_input.html